README.md

```diff
@@ -722,6 +722,15 @@ We recommend you specify exact versions of lint libraries, including `tslint-mic
       </td>
       <td>4.0.2</td>
     </tr>
+    <tr>
+      <td>
+        <code>void-zero</code>
+      </td>
+      <td>
+        <code>void 0</code>, which resolves to <code>undefined</code>, can be confusing newcomers. Exclusively use <code>undefined</code> to reduce ambiguity.
+      </td>
+      <td>6.0.0</td>
```

Comment:
`6.0.0` is current verstion. This rule will be added in next minor version - `6.1.0`.

Reply:
Ah. Thanks

Reply:
Oh I thought you were linking to this when you said one more question. Define `fixer` :)